hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSConfigKeys.java

```diff
@@ -276,6 +276,10 @@ public class DFSConfigKeys extends CommonConfigurationKeys {
   public static final boolean
       DFS_NAMENODE_REDUNDANCY_CONSIDERLOADBYVOLUME_DEFAULT
       = false;
+  public static final String DFS_NAMENODE_CHOOSEREPLICATODELETE_CONSIDERLOAD_KEY =
+      "dfs.namenode.choosereplicatodelete.considerLoad";
```

Comment:
nit: could we change to "dfs.namenode.load-based.replica.deletion"?

Reply:
@xinglin Thanks bro.  How about "dfs.namenode.replica.deletion.considerLoad" ？ What's your opinions？

Reply:
"considerLoad" is what make me feel awkward. 

Maybe "dfs.namenode.replica.deletion.by.load", "dfs.namenode.replica.deletion.based-on-load", "dfs.namenode.replica.deletion.by.minimal.load", or "dfs.namenode.replica.deletion.load-based".